resync/workflows/workflow_predictive_maintenance.py

```
@@ -629,14 +747,80 @@ def create_predictive_maintenance_workflow(
+    """Validated request for workflow approval."""
+
+    workflow_id: str = Field(..., max_length=100)
+    approved: bool
+    feedback: str | None = Field(default=None, max_length=5000)
+    user_id: str = Field(..., min_length=1, max_length=100)
+
+    @field_validator('workflow_id')
+    @classmethod
+    def validate_workflow_id(cls, v: str) -> str:
+        if not re.match(r'^pm_[a-zA-Z0-9_\-\.]+_[a-f0-9]+$', v):
+            raise ValueError("workflow_id format invalid")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)